### PR TITLE
Testing branch to merge with main

### PR DESCRIPTION
**Pull request by TechnicallyRy**
Pull request for merge to main.

### CODE REVIEW

File: lib/fifo/src/fifo.c [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        //data manipulation
+        data.output = data.input + 5;
+        data.handled_by = id;
+
+        k_msgq_put(results, &data, K_FOREVER);  //K_FOREVER: keep trying to put data in buffer until it can
+
+    }
+
+
+
```
Comment (compugal99) [inline lib/fifo/src/fifo.c:19]: I tell students this as a TA for CS 1420 but avoid having tons of whitespace like this

File: platformio.ini [hunk 2]
```diff
@@ -8,7 +8,30 @@
...
+[env:disco_board]
+monitor_speed = 115200
+
+[env:disco_renode]
+platform_packages =
+    platformio/tool-renode
+test_testing_command =
+    ${platformio.packages_dir}/tool-renode/renode
+    --disable-xwt
+    --console
```
Comment (compugal99) [inline platformio.ini:28]: I like the way you have divided up the environment for the testing and uploading.

**Review by compugal99 — APPROVED** [review]
Just left a couple of suggestions. The code looks really good.

**Review by willgraham345 — APPROVED** [review]
Well done guys! Sorry for the late response, didn't realize I had this outstanding until Alex responded. 

One thing I'd suggest is making sure your spacing is uniform throughout the document. That's really specific so I wouldn't worry about it much.

**Pull request merged**